DiceKit/FrequencyDistribution.swift

```diff
@@ -222,9 +239,25 @@ extension FrequencyDistribution {
         }
     }
     
-    // TODO: public func divide(x: FrequencyDistribution) -> FrequencyDistribution
-    // Needed to support power(-n)
-    
+    public func divide(y: FrequencyDistribution) -> FrequencyDistribution {
+        guard let initialK = orderedOutcomes.first, lastK = orderedOutcomes.last else {
+            return .additiveIdentity
+        }
+        guard let firstY = y.orderedOutcomes.first, lastY = y.orderedOutcomes.last, firstYFrequency = y[firstY] where firstYFrequency != 0.0 else {
+            fatalError("Divide by zero! :(")
```

Comment:
D: 

I can write a better message. Should should I stick with using `fatalError`?